pom.xml

```diff
@@ -63,6 +73,20 @@
 					</excludes>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.11.0</version>
+				<configuration>
+					<annotationProcessorPaths>
```

Comment:
element annotationProcessorPath powinien wyglądać następująco
```
	<annotationProcessorPaths>
		<path>
			<groupId>org.projectlombok</groupId>
			<artifactId>lombok</artifactId>
			<version>1.18.30</version>
		</path>
		<path>
			<groupId>org.mapstruct</groupId>
			<artifactId>mapstruct-processor</artifactId>
			<version>1.5.5.Final</version>
		</path>
		<path>
			<groupId>org.projectlombok</groupId>
			<artifactId>lombok-mapstruct-binding</artifactId>
			<version>0.2.0</version>
		</path>
	</annotationProcessorPaths>

```


wynika to z faktu że zarówno mapstruct jak i lombok korzystając z annotationProcessora i istotna jest tutaj kolejność tych deklaracji. Mapper nie mógł zadziałać, ponieważ w procesie mvn był wywoływany zanim klasy zostały stworzone przez lombok. więcej możesz poczytać [tutaj]( https://bootify.io/spring-data/mapstruct-with-maven-and-lombok.html) a na spotkaniu opowiem ci jak do tego doszłam